src/Components/daterange/NavBarRange.svelte

```diff
@@ -0,0 +1,250 @@
+<script>
+  import { createEventDispatcher } from 'svelte';
```

Comment:
Believe this can be mixed into https://github.com/6eDesign/svelte-calendar/blob/master/src/Components/NavBar.svelte with a flag, or check on props.